packages/model/index.js

```diff
@@ -42,6 +42,7 @@ module.exports = Object.assign({}, addonBaseConfig, {
       'ember-inflector',
       'ember',
       'rsvp',
+      '@embroider/macros/runtime',
```

Comment:
This is only needed in development builds. In production builds, the macros have no runtime implementation and get compiled down to constants.

Reply:
Oh, and I only updated this one because it's the only case I hit while I was testing, and I was only testing against `packages/-ember-data`. This might need to go into other packages if there are other similar errors.